tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+@pytest.fixture
+def enterCredentials():
+    print("To run the examples, enter your credentials for the GTI API.")
+    gti_user = input("GTI Username: ")
```

Comment:
Don't actually call the API for tests, use "unittest.mock.patch" to mock an API response